lib/travis/build/addons/homebrew.rb

```diff
@@ -72,7 +72,7 @@ def user_brewfile?
         end
 
         def brew_bundle_args
-          if config[:brewfile] == true
+          if config[:brewfile].to_s.downcase == 'true'
             ''
           else
```

Comment:
So, the unlikely scenario where user specifies the brewfile named `true` is ignored. Is it worth documenting?

Reply:
I'd be surprised if someone actually did this in practice, so I don't think it's worth documenting.